core/src/main/java/org/kiwix/kiwixmobile/core/help/HelpFragment.kt

```diff
@@ -88,16 +89,21 @@ abstract class HelpFragment : BaseFragment() {
   }
 
   private fun sendFeedback() {
-    val intent = Intent(Intent.ACTION_SENDTO)
-    intent.data = (
-      "mailto:${Uri.encode(CONTACT_EMAIL_ADDRESS)}" +
-        "?subject=" +
-        Uri.encode(
-          "Feedback in " +
-            getCurrentLocale(requireActivity()).displayLanguage
-        )
-      ).toUri()
-    startActivity(Intent.createChooser(intent, "Send Feedback via Email"))
+    val emailIntent = Intent(Intent.ACTION_SENDTO).apply {
+      data = (
+        "mailto:${Uri.encode(CONTACT_EMAIL_ADDRESS)}" +
+          "?subject=" +
+          Uri.encode("Feedback in ${getCurrentLocale(requireActivity()).displayLanguage}")
+        ).toUri()
+    }
+    val packageManager = requireActivity().packageManager
+    val activities = packageManager.queryIntentActivities(emailIntent, 0)
+    if (activities.isNotEmpty()) {
+      val chooserIntent = Intent.createChooser(emailIntent, "Send Feedback via Email")
```

Comment:
Since ```chooserIntent``` is used only once, it is recommended to use it as an inline variable instead of declaring it separately in order to avoid writing an extra line of code, like we previously using.